component/common/src/main/java/org/wso2/carbon/extension/identity/emailotp/common/dto/SessionDTO.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.extension.identity.emailotp.common.dto;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Object used to save the data in the database
+ */
+
```

Comment:
Addintional new line.

Reply:
Fixed in af803a284224d961eb0a744e751122f917b9b0af